front/src/pods/player/player.component.tsx

```diff
@@ -146,7 +146,7 @@ const ShowVotingResultsComponent: React.FC<ShowVotingResultsProps> = props => {
           />
         </div>
         <div className={classes.containerComponent}>
-          <h2 className={classes.title}>Wait for next story</h2>
```

Comment:
Remove h2 component with text `Show voting results` line [140](https://github.com/Lemoncode/planning-poker-example/pull/112/files#diff-cfde6183bcd22952f17e0bd765c1f25113b7b9e4957c356b83336722605acf20R140) as comment on point 7:

`7. Cuando el estudiante ve los resultados, quitar el texto Show voting results (comenta que no aporta nada), dejar tal y como está el h3 de "Players connected"`

Reply:
This is a mistake? This is not a Show voting results....

It's in the same component but up in the code.

I fix that code...

`
<div className={classes.rightContainer}>
        <div className={classes.containerComponent}>
//I will delete this H2
          <h2 className={classes.title}>Show voting results</h2>
        </div>
        <div className={classes.containerComponent}>
          <TablePlayerComponent
            playersCollection={voteCollectionResult}
            headingLevel="h3"
          />
        </div>
        <div className={classes.containerComponent}>
          <p className={classes.title}>Wait for next story</p>
        </div>
      </div>

`

Reply:
You are right, it above the line code in line 140, but Github doesn't allow write comments in collapsed lines. Thank you for the update